modules/images-describing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-describing-image-registry-settings-hcp_{context}"]
+= Describing image registry settings for ROSA with HCP
+
+
+You can block any registry, and optionally an individual repository within a registry, by editing the `image.config.openshift.io/cluster` custom resource (CR). {product-title} applies the changes to this CR to all nodes in the cluster.
+
+When pulling or pushing images, the container runtime searches the registries listed under the `registrySources` parameter in the `image.config.openshift.io/cluster` CR. If you created a list of registries under the `blockedRegistries` parameter, the container runtime does not search those registries. All other registries are allowed.
+
+[WARNING]
+====
+To prevent pod failure, do not add the `registry.redhat.io` and `quay.io` registries to the `blockedRegistries` list, as they are required by payload images within your environment.
+====
+
+.Procedure
+
+. Describe ROSA with HCP clusters with image registry:
++
+[source,terminal]
+----
+$ rosa describe cluster -c <cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+Name:                       magchen-hcp
```

Comment:
You can copy the describe example from https://82728--ocpdocs-pr.netlify.app/openshift-rosa/latest/rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth
Replace `External Authentication:    Enabled ` with 
```
Registry Configuration:
 - Allowed Registries:       <allowed_registries><1>
 - Insecure Registries:     <insecure_registries><2>
 - Allowed Registries for Import: <3>
    - Domain Name:          quay.io
    - Insecure:             true
```